src/bindings/c/include/openvino/c/gpu_plugin_properties.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
Can we have a plugin dedicated folders, like we have in cpp API?
Something like: `openvino/c/gpu/`

Reply:
ok

Reply:
done